labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise02.scala

```diff
@@ -27,7 +32,7 @@ object ListManipulationExercise02 {
    * method above
    */
   def sumOfMany(l: List[Int]*): List[Int] = {
-    error("fix me")
+    l reduce sumOfTwo
```

Comment:
What kind of unit testing doesn’t *test* the empty list, anyway? Any CS70er who’s done ChunkyString can probably write better tests.